dapp/src/components/LedgerDerivationContent.js

```diff
@@ -38,30 +65,143 @@ const LedgerDerivationContent = ({}) => {
 
   const options = [
     {
-      display: `Ledger Live - m/${LEDGER_LIVE_BASE_PATH}`,
+      display: `Ledger Live`,
       path: LEDGER_LIVE_BASE_PATH,
     },
     {
-      display: `Legacy - m/${LEDGER_LEGACY_BASE_PATH}`,
+      display: `Legacy`,
       path: LEDGER_LEGACY_BASE_PATH,
     },
     {
-      display: `Ethereum - m/${LEDGER_OTHER}`,
+      display: `Ethereum`,
       path: LEDGER_OTHER,
     },
   ]
 
-  const onSelectDerivationPath = async (path) => {
+  const loadBalances = async (path) => {
+    if (ledgerConnector.provider && addresses[path]) {
```

Comment:
You can early exit a function by negating a statement and avoiding `if` indentation. Changing this:  
```
if(something) {
  // a lot of lines of execution
}
```
to this:
```
if(!something) {
  return;
}
// a lot of lines of execution (not indented) 
```

Reply:
Oh another nitpick when arrow is facing upwards it is not centered: 

<img width="114" alt="Screenshot 2022-03-18 at 11 18 18" src="https://user-images.githubusercontent.com/579910/158992483-61e48fed-c10f-4d3b-998e-1a06d69b064e.png">